USER_MANUAL.md

```diff
@@ -628,7 +673,526 @@ The measured total site level emission rate on January 30th is assumed to have l
+
+**Description:** An integer indicating the number of required surveys at each facility per calendar year.
+
+**Notes on acquisition:** Survey frequencies can be based on regulatory requirements, company policies, or can be fabricated by the modeler to explore different scenarios.
+
+**Notes of caution:** Note that just because a number of surveys is prescribed, it does not mean that this number of surveys will necessarily be performed. For example, if labour limitations exist (i.e., not enough crews are available to inspect the number of facilities in the program) or if environmental conditions are unsuitable (i.e., a particular facility is in a cloudy location that cannot be accessed by satellite), the performed number of surveys may be less than the prescribed number. This variable is not required for continuous measurement methods.
+
+**Note:** this parameter may also be set more granularly using the infrastructure files.
+
+### &lt;survey_time&gt; _(propagating parameter)_
+
+**Data type:**  Numeric (Integer)
+
+**Default input:** N/A
+
+**Description:** The number in minutes required to complete a survey or screening at each facility.
+
+**Notes on acquisition:** In most cases, an estimate will be made as data will not exist for the specific combination of facility and unique method. However, as new methods and programs are implemented, data will become available to better refine modeling estimates and develop more intelligent programs.
+
+**Notes of caution:** This variable is an empirical estimate of how much time is required for a given mobile method to complete a survey at a given facility. This includes anything that happens onsite (e.g., calibrations, interfacing with the operator, etc.) but _does not include_ driving time between facilities or any other account of time spent offsite. This variable is simply the amount of time that passes from the start of a facility survey to the end. If a facility takes longer than there is time left in a day, then the agent/crew returns the following day to continue work, and so on and so forth, until the facility is completed. This variable is not required for continuous measurement methods.
+
+**Note:** this parameter may also be set more granularly using the infrastructure files.
+
+### &lt;max_workday&gt;
+
+**Data type:**  Numeric (Integer)
+
+**Default input:** 8
+
+**Description:** The maximum number of hours a crew can work in day (includes travel time).
+
+**Notes on acquisition:** No data acquisition required.
+
+**Notes of caution:** This can be overridden if [consider_daylight](#consider_daylight) is True and the valid daylight hours are shorter.
+
+### &lt;reporting_delay&gt;
+
+**Data type:** Numeric (Integer)
+
+**Default input:** 2
+
+**Description:** The number of days that pass between the end of a survey (when a site is flagged or emissions are tagged) and when the duty holder is informed. The reporting delay is then followed by the [repair delay](#delay).
```

Comment:
Nit: Reporting delay is only followed by repair delay when the method is component level. Perhaps we can adjust the wording here to be more generally applicable.